react/features/base/conference/middleware.web.js

```diff
@@ -21,3 +26,34 @@ MiddlewareRegistry.register((/* store */) => next => action => {
 
     return next(action);
 });
+
+/**
+ * Notifies the feature base/conference that the action {@code CONFERENCE_RESTARTED}
+ * is being dispatched within a specific redux store.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action {@code CONFERENCE_RESTARTED}
+ * which is being dispatched in the specified {@code store}.
+ * @private
+ * @returns {Object} The value returned by {@code next(action)}.
+ */
+function _conferenceRestarted({ dispatch, getState }, next, action) {
+    const { enableForcedReload } = getState()['features/base/config'];
+
+    if (enableForcedReload) {
+        const reason = 'Client reload initiated because of a bridge failure';
+
+        dispatch(showErrorNotification({
+            description: reason,
+            titleKey: 'dialog.sessionRestarted'
+        }));
+
+        // Make sure pre-join page is skipped when client is reloaded.
+        dispatch(setSkipPrejoinOnReload(true));
```

Comment:
I'm wondering, shouldn't we skip the prejoin on every reload triggered by jitsi-meet itself? 

As part of the `reloadNow` action we add the current muted state for audio and video in order to preserve them. Also the selected devices and the display name should be the same. So I guess the prejoin screen isn't that useful in those situations.

WDYT?

Reply:
Maybe ?  We can check with the rest of the team before making that happen for all other cases of forced reload.

Reply:
Sounds good!